site/content/blog/2021-02-03-release-0_8_0.md

```diff
@@ -0,0 +1,100 @@
+
+# New Maintainers
+The biggest news we have, and probably one of the best things for the Trunk project itself, is that we have 3 new maintainers!!! 🎉🍻 Rakshith Ravi, Philip Peterson and Martin Kavik. I'm very excited to have these guys on the team, they've each already contributed in many ways to the Trunk project, and having a strong group of maintainers is very important for the long-term success of any open source project. We are all stronger together!
+
+If you are interested in helping out as a maintainer, the best way to get started is with PR reviews, reviewing issues and making sure they are ready to be moved to the coding phase, and then actually getting your hands dirty with some code! Trunk so far has been able to maintain a pretty solid and enjoyable code base. Now is a great time to get involved.
+
+# Trunk v0.8.0 has Landed
+Trunk v0.8.0 has now been released. See the [release page](https://github.com/thedodd/trunk/releases/tag/v0.8.0) for all of the details. In summary:
+
+- The `trunk watch` & `trunk serve` commands now accept the `--watch` option which allows users to specify multiple different directories and files to be watched, triggering new builds as changes are detected. A community member `@malobre` knocked this one out.
+- Another community member `@DzenanJupic` has added the `inline` asset type. This allows users to target various file types and have their content "inlined" into the parent HTML document as-is. Go checkout the docs on the [inline asset type](https://trunkrs.dev/assets/#inline) for more details (wow, felt great to be able to provide a link to the docs like that).
+- A community member `@hamza1311` & one of our maintainers `@philip-peterson` squashed a bug where old build artifacts would continually accumulate in the dist dir. That has now been fixed, and any time a successful build goes through, all of the old artifacts will be removed, and only the fresh new artifacts will exist in the `dist` dir.
+- Lastly, `@MartinKavik` fixed an infinite rebuild loop bug on Windows.
+
+# Immediate Plans
+In the immediate term, we have a few community lead efforts for cutting Trunk over to the Tokio ecosystem. This is primarily to address our need for a robust WebSockets & SSE solution, and the Tokio community happens to have some pretty solid options on that front.
+
+We've also got another community lead effort on getting `wasm-opt` integrated into the standard Trunk build pipeline.
+
+Both of these initiatives are exciting and will go far to provide an even better experience for folks building Rust WASM application using Trunk.
+
+# WASM Plugin Architecture for Trunk
```

Comment:
The blog post seems too long to me and the section `# WASM Plugin Architecture for Trunk` feels more like a part of the documentation than content for a blog post. I would move that section to a standalone page.

Reply:
That's good feedback. I'll go ahead and pull that out as a second blog post. I definitely agree that having it in some "architecture" docs somewhere would be good, but for the sake of the discussion on a plugin system, I'll keep it in the post and then we can copy it to a more formal docs location in the future.